### ISSUE TITLE
When sourcemap tool runs out, our "ready when you are" page looks wrong 

### ISSUE THREAD

**Issue by jonbell-lot23** [RecordReplay/devtools#5447]
<img width="465" alt="image" src="https://user-images.githubusercontent.com/9154902/155430855-53e11b39-5ece-42ea-90b9-97c80b14b14e.png">

**Comment by jasonLaster** [comment]
Agreed - we shouldn't show that in this context. Which is similar to the library

**Comment by jonbell-lot23** [comment]
Jason fixed this so I'm taking myself off the ticket
